front-end/src/App.js

```diff
@@ -42,6 +42,7 @@ function App() {
   }
 
   useEffect(async () => {
```

Comment:
I think in the future we should move all the functions related to the authentication in this useEffect to its own file. App.js files are typically very simple and make it easy to see the final output of the app. With all the functions located in this file it takes much more time to parse and the functionality could be potentially reused later in the app if applicable.

For now this is okay, as it doesn't affect functionality, but I've made an issue #78 so we can update this in the near future.